tests/update_workflow.go

```diff
@@ -180,6 +180,15 @@ func (s *FunctionalSuite) pollUpdate(tv *testvars.TestVars, updateID string, wai
 	})
 }
 
+func (s *FunctionalSuite) clearWorkflowContext(tv *testvars.TestVars) *adminservice.DescribeMutableStateResponse {
+	resp, err := s.adminClient.DescribeMutableState(NewContext(), &adminservice.DescribeMutableStateRequest{
```

Comment:
Can you add a comment here, explaining why `DescribeMutableState` is used for this. Similar to ones I had before:
```
// Side effect of DescribeMutableState is clear of workflow context because
// it needs to return both cached and database versions.
```